Content.Server/Cloning/CloningSystem.cs

```diff
@@ -86,6 +73,8 @@ public override void Initialize()
         {
             base.Initialize();
 
+            _sawmill = _log.GetSawmill("cloning"); // DeltaV
```

Comment:
why do this instead of Log.Error which uses system.cloning

Reply:
fair, leftover from another implementation before i had the genius idea to do this